src/ai/backend/manager/api/vfolder.py

```diff
@@ -3004,26 +3004,23 @@ async def change_vfolder_ownership(request: web.Request, params: Any) -> web.Res
     Change the ownership of vfolder
     For now, we only provide changing the ownership of user-folder
     """
-    access_key = request["keypair"]["access_key"]
     vfolder_id = params["vfolder"]
-    user_uuid = params["user"]
+    user_email = params["user_email"]
     root_ctx: RootContext = request.app["_root.context"]
 
     # TODO: Implement Change ownership using DB transaction
     log.info(
-        "VFOLDER.CHANGE_VFOLDER_OWNERSHIP(email:{}, ak:{}, vfid:{}, uid:{})",
-        request["user"]["email"],
-        access_key,
+        "VFOLDER.CHANGE_VFOLDER_OWNERSHIP(new owner email:{}, vfid:{})",
+        user_email,
```

Comment:
Let's remain the logging of requester's info! we can add new logging of "new owner"

Reply:
Oh It's doesn't reflects the original "owner", It's the user(actually, super admin) who requested.